lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -1834,9 +1841,20 @@ spv_ostream &operator<<(spv_ostream &O, SPIRVModule &M) {
   O << SPIRVMemoryModel(&M);
 
   for (auto &I : MI.EntryPointVec)
-    for (auto &II : I.second)
+    for (auto &II : I.second) {
+      // getVariables() will return a non-empty vector only when translating
+      // LLVM-IR -> SPIRV text, because the entry points was translated as
+      // SPIRVFunction and their interfaces are saved as variables by
+      // collectEntryPointInterfaces function. When translating SPIRV bitcode ->
+      // SPIRV text the entry points are decoded from the bitcode and the
+      // interfaces are stored to EntryPointInterfaces map to not to be lost.
```

Comment:
Why do we need extra containers in `SPIRVModule` and `SPIRVFunction` classes to store interface ids? I mean can we store the ids in `SPIRVEntryPoint::Variables`?